packages/core/core/src/Parcel.js

```diff
@@ -71,10 +70,10 @@ export default class Parcel {
       this.#initialOptions
     );
     this.#resolvedOptions = resolvedOptions;
-    await createCacheDir(resolvedOptions.cacheDir);
+    await createCacheDir(resolvedOptions.outputFS, resolvedOptions.cacheDir);
 
     let {config} = await loadParcelConfig(
-      path.join(process.cwd(), 'index'),
+      path.join(resolvedOptions.inputFS.cwd(), 'index'),
```

Comment:
what's the rationale for moving `cwd` to the filesystem interface? Any time we use a different fs backend we'll also want to implement `cwd` differently?

Reply:
`process.cwd()` is implicitly referencing the native OS filesystem. There is no guarantee that this directory will exist in other filesystems.